internal/cli/auth/login.go

```diff
@@ -178,7 +178,7 @@ func LoginBuilder() *cobra.Command {
 		Use:   "login",
 		Short: "Authenticate with MongoDB Atlas.",
 		Example: `  To start the interactive setup:
-  $ mongocli auth login
+  $ atlas auth login
```

Comment:
this doesn’t work as it adds `atlas` also to mongocli. you have to use` fmt.Sprintf(`) combined with `config.BinName()`

Reply:
Thank you! Didn't think of that.